### PR TITLE
Rename public api fields from _uuid to _id to be consistent with internal API

### PR DESCRIPTION
**Pull request by chrisgilmerproj**
## Description

The internal api doesn't specify `_uuid` and instead uses `_id`.  We've only partially changed the public api to be consistent and this PR goes the rest of the way towards getting it done.
## Code Review Verification Steps

* [x] Request review from a member of a different team.

### CODE REVIEW

File: swagger/api.yaml [hunk 1]
```diff
@@ -852,15 +852,15 @@ paths:
           description: not authorized to create blackouts
         500:
           description: server error
-  /blackouts/{blackout_uuid}:
+  /blackouts/{blackout_id}:
```
Comment (kilbergr) [inline swagger/api.yaml]: if I were you, I would change this to the style blackoutId. That's how it's done in the internal API, so I don't see much value to updating to make it more similar but still not quite the same.

File: swagger/api.yaml [hunk 2]
```diff
@@ -210,7 +210,7 @@ definitions:
         type: string
         format: uuid
         example: c56a4180-65aa-42ec-a945-5fd21dec0538
-      shipment_uuid:
+      shipment_id:
```
Comment (kilbergr) [inline swagger/api.yaml]: good.

**Comment by chrisgilmerproj** [comment]
Alright, I think this fixes it up.  I hadn't realized we did `Id` instead of `_id` and probably should have looked before pushing this out for review.  

I've also gone in and tried to update a few other places where that `_id` or `_uuid` existed in tests and docs and removed or fixed them.

File: swagger/api.yaml [hunk 3]
```diff
@@ -94,7 +94,7 @@ definitions:
         description: unique id of the blackout in the system
         format: uuid
         example: c56a4180-65aa-42ec-a945-5fd21dec0538
-      tsp_id:
+      tspId:
```
[the remark this answers is not in the document]
Comment (chrisgilmerproj) [reply swagger/api.yaml]: That is ridiculous! 😛 but that's ok.  I'll do another pass at this and I'm glad you caught it.

File: pkg/handlers/publicapi/service_agents_test.go [hunk 4]
```diff
@@ -108,7 +109,7 @@ func (suite *HandlerSuite) TestPatchServiceAgentHandler() {
 	serviceAgent := serviceAgents[0]
 
 	// And: the context contains the auth values
-	req := httptest.NewRequest("PATCH", "/shipments/shipment_id/service_agents/service_agents_id", nil)
+	req := httptest.NewRequest("PATCH", "/shipments/shipmentId/service_agents/service_agentsId", nil)
```
[the remark this answers is not in the document]
Comment (chrisgilmerproj) [reply pkg/handlers/publicapi/service_agents_test.go]: Aha! My regex failed me yet again!

**Comment by chrisgilmerproj** [comment]
🎉

**Pull request merged**